package.xml

```diff
@@ -7,6 +7,12 @@
   <maintainer email="william@osrfoundation.org">William Woodall</maintainer>
   <license>Apache License 2.0</license>
 
+  <build_depend>python3-setuptools</build_depend>
+
+  <buildtool_depend>python3</buildtool_depend>
+
+  <buildtool_export_depend>python3</buildtool_export_depend>
```

Comment:
Yeah, I'll admit that I cargo-culted these keys from ament_package, which was the first ament_python package, when the next set of packages failed to build. The buildtool depend on python is required for pybuild/dh_python3 support. The export depend might not be necessary.